numba/decorators.py

```diff
@@ -246,3 +251,25 @@ def wrapper(func):
         return res
 
     return wrapper
+
+
+def jit_module(**kwargs):
+    """ Automatically ``jit``-wraps functions defined in a Python module
+
+    Note that ``jit_module`` should only be called at the end of the module to
+    be jitted. In addition, only functions which are defined in the module 
+    ``jit_module`` is called from will be automatically jitted.
```

Comment:
I'm wondering if this should be something like:

> In addition, only functions which are defined in the module  ``jit_module`` is called from are considered for automatic jit-wrapping. See the Numba documentation for more information about what can/cannot be jitted.

so as to hint at the caveats?